apps/web/src/components/TrendingSuggestions/TrendingSuggestions.tsx

```diff
@@ -0,0 +1,32 @@
+import * as React from 'react';
+import { useQuery } from 'react-query';
+import fetchTrendingSuggestions from './TrendingSuggestions.api';
+import TrendingSuggestionList from '../TrendingSuggestionList/TrendingSuggestionList';
+
+const TrendingSuggestions = () => {
+  const { data, isLoading } = useQuery(
+    ['trending-suggestions'],
+    () => {
+      const response = fetchTrendingSuggestions();
+      return response;
+    },
+    { staleTime: Infinity }
+  );
+
+  // placeholder func, will be removed when we add react router + refactor
```

Comment:
In a comment like this that references tech debt, it's always a good idea to link to the ticket that fixes the issue.